Gruntfile.js

```diff
@@ -90,7 +90,7 @@ module.exports = function(grunt) {
         options: {
           // The src/vendor paths are needed to find the CF components' files.
           // Feel free to add additional paths to the array passed to `concat`.
-          paths: grunt.file.expand('src/vendor/*').concat([])
+          paths: '<%= loc.src %>/vendor/'
```

Comment:
Can you be more specific about what you're doing here and why? Also, if this change remains, the comment above is out of date.